docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.3 Release Notes
+description: Release notes for NuGet 6.3 including new features, bug fixes, and DCRs.
+author: <GithubAlias>
+ms.author: <MicrosoftAlias>
+ms.date: 8/2/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.3 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+
+## Summary: What's New in 6.3
+
+* Implement "consuming pdbs from packages in PackageReference" - [#11352](https://github.com/NuGet/Home/issues/11352)
+
+* Create experimentation metrics to measure engagement of transitive dependencies - [#10759](https://github.com/NuGet/Home/issues/10759)
+
+* Enhance the experimentation infrastructure in NuGet code to support transitive dependencies - [#10758](https://github.com/NuGet/Home/issues/10758)
```

Comment:
Same, I believe this was mislabeled as a feature. It's part of a feature, so probably should be tagged as a bug, a code change.

Reply:
Moving this to the bottom of the list